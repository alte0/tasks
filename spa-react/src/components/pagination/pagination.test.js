import React from 'react';
import renderer from 'react-test-renderer';
import Pagination from './pagination';

it('Pagination render', () => {
    const tree = renderer
        .create(
            <Pagination pagesCount={9} pageCurrentPagination={0}/>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
