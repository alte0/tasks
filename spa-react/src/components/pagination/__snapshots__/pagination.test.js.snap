// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination render 1`] = `
<ul
  className="pagination-list"
>
  <li
    className="pagination-list__item pagination-list__item_prev"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={-1}
      href="page--1"
    >
      &lt;
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={1}
      href="page-1"
    >
      1
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={2}
      href="page-2"
    >
      2
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={3}
      href="page-3"
    >
      3
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={4}
      href="page-4"
    >
      4
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={5}
      href="page-5"
    >
      5
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={6}
      href="page-6"
    >
      6
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={7}
      href="page-7"
    >
      7
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={8}
      href="page-8"
    >
      8
    </a>
  </li>
  <li
    className="pagination-list__item"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={9}
      href="page-9"
    >
      9
    </a>
  </li>
  <li
    className="pagination-list__item pagination-list__item_next"
  >
    <a
      className="pagination-list__link"
      data-page-id-pag={1}
      href="page-1"
    >
      &gt;
    </a>
  </li>
</ul>
`;
